sdk/communication/Azure.Communication.Chat/api/Azure.Communication.Chat.netstandard2.0.cs

```diff
@@ -13,9 +13,9 @@ public partial class AddChatParticipantsResult
     public partial class ChatClient
     {
         protected ChatClient() { }
-        public ChatClient(System.Uri endpointUrl, Azure.Communication.CommunicationTokenCredential communicationTokenCredential, Azure.Communication.Chat.ChatClientOptions? options = null) { }
-        public virtual Azure.Response<Azure.Communication.Chat.CreateChatThreadResult> CreateChatThread(string topic, System.Collections.Generic.IEnumerable<Azure.Communication.Chat.ChatParticipant> participants, string? repeatabilityRequestId = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Communication.Chat.CreateChatThreadResult>> CreateChatThreadAsync(string topic, System.Collections.Generic.IEnumerable<Azure.Communication.Chat.ChatParticipant> participants, string? repeatabilityRequestId = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public ChatClient(System.Uri endpointUrl, Azure.Communication.CommunicationTokenCredential communicationTokenCredential, Azure.Communication.Chat.ChatClientOptions options = null) { }
```

Comment:
`endpointUrl` should be called `endpoint`

Reply:
this one is on the list for preview5, no hurt in doing now if we want.

Reply:
perhaps let's do the changes asked from the azure review together?

Reply:
Is not it better to just do it instead of having the same comment on another PR? + and one less thing to worry in the future 😉